Examples/UseCases.ipynb

```diff
@@ -129,7 +144,16 @@
    "source": [
     "from pydkpro import DKProCoreTypeSystem as dts\n",
     "\n",
-    "cas.select(dts().token).as_text() "
+    "cas.select(dts().token).as_text() \n",
+    "\n",
+    "// REC: I wonder which version of the DKProCoreTypeSystem is used here...\n",
+    "\n",
+    "// REC: Why is dts() a method? It would seem quite redundant to always have to call this method.\n",
+    "// At least to me, it would feel more natural to write something like this:\n",
+    "    \n",
+    "from pydkpro.typesystem import Token\n",
```

Comment:
If we use ```from pydkpro.typesystem import token``` (small case for Token), then  ```cas.select(token).as_text()```  should work. I'll update it like that.

Reply:
But token is a type, so it should be upper-case `Token`, no?